src/cascadia/TerminalSettingsModel/GlobalAppSettings.cpp

```diff
@@ -94,11 +94,7 @@ winrt::com_ptr<GlobalAppSettings> GlobalAppSettings::Copy() const
     }
     if (_DisabledProfileSources)
     {
-        globals->_DisabledProfileSources = winrt::single_threaded_vector<hstring>();
-        for (const auto& src : *_DisabledProfileSources)
-        {
-            globals->_DisabledProfileSources->Append(src);
-        }
+        globals->_DisabledProfileSources = winrt::single_threaded_vector<hstring>(std::move(wil::to_vector(*globals->_DisabledProfileSources)));

```

Comment:
FYI: A returned by-value object is being-moved, so moving it again doesn't do anything. (In fact it makes it worse, because it takes a reference, so you force the return value on the stack - not a big issue tho.)